docs/website/versioned_docs/version-3.0.0/user-guides/quickstart.md

```diff
@@ -112,6 +147,46 @@ Changes will be directly reflected on the cluster.
 A `devfile.yaml` has now been added to your directory and now you're ready to start development.
 
   </TabItem>
+  <TabItem value="dotnet" label=".NET">
+
+Let's run `odo init` and select .NET 6.0:
+
+```console
+$ odo init
+  __
+ /  \__     Initializing new component
+ \__/  \    Files: Source code detected, a Devfile will be determined based upon source code autodetection
+ /  \__/    odo version: v3.0.0-alpha2
+ \__/
+
+Interactive mode enabled, please answer the following questions:
+Based on the files in the current directory odo detected
+Language: javascript
+Project type: nodejs
+The devfile "nodejs" from the registry "DefaultDevfileRegistry" will be downloaded.
```

Comment:
I guess it depends on Alizer, but it is weird to me that "nodejs" was detected, and not "dotnet"

Reply:
I know :( Alizer wasn't able to detect it correctly. I'll be opening up an issue for this.

Reply:
FYI, I just found a similar issue: https://github.com/redhat-developer/alizer/issues/83